fastexcel-reader/src/main/java/org/dhatim/fastexcel/reader/OPCPackage.java

```diff
@@ -162,7 +163,18 @@ private InputStream getEntryContent(String name) throws IOException {
         }
         ZipArchiveEntry entry = zip.getEntry(name);
         if (entry == null) {
-            return null;
+            // to be case insensitive
+            Enumeration<ZipArchiveEntry> entries = zip.getEntries();
+            while (entries.hasMoreElements()) {
+                ZipArchiveEntry e = entries.nextElement();
+                if (e.getName().equalsIgnoreCase(name)) {
+                    entry = e;
```

Comment:
I suggest to return zip.getInputStream(e) here and remove condition after loop and directly return null if we are out of this loop without returning

Reply:
fixed